From 6b621b768e41b6ab13d2b6589e7e5abf04711b5b Mon Sep 17 00:00:00 2001
From: Tejas Patel <tejas.patel1@einfochips.com>
Date: Wed, 19 Jan 2022 13:22:00 +0530
Subject: [PATCH 06/13] arch: arm64: configs: Enable AP1302 module in imx
 defconfig

This patch enables AP1302 as loadable module in imx_v8_defconfig.

Change-Id: I2e38ceb0f195f431776bde847ce8fc4eb5538a08
Signed-off-by: Tejas Patel <tejas.patel1@einfochips.com>
Signed-off-by: Tanvi Chauhan <tanvi.chauhan@einfochips.com>
---
 arch/arm64/configs/imx_v8_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index c5879e93d..ecfc7efac 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -525,6 +525,7 @@ CONFIG_MXC_MIPI_CSI=y
 CONFIG_MXC_CAMERA_OV5640_MIPI_V2=y
 CONFIG_V4L_MEM2MEM_DRIVERS=y
 CONFIG_SDR_PLATFORM_DRIVERS=y
+CONFIG_VIDEO_AP1302=m
 CONFIG_VIDEO_IMX219=m
 CONFIG_VIDEO_OV5640=y
 CONFIG_VIDEO_OV5645=m
-- 
2.17.1

